data_validation/__main__.py

```diff
@@ -479,9 +480,9 @@ def main():
     elif args.command == "configs":
         run_validation_configs(args)
     elif args.command == "find-tables":
-        print(find_tables_using_string_matching(args))
+        logging.info(find_tables_using_string_matching(args))
```

Comment:
Also, we could add a variable coming out of a config file to for level in - logging.basicConfig(level=logging.INFO, format='%(asctime)s-%(levelname)s: %(message)s'). This will help us change it to different Level without any code changes.

Reply:
I'm worried about this change.  This is not actually a log, it is the output of the command.

Personally, I use this response to build validations for a whole schema when i know the names will be very similar.

In reality, my usage should be a feature which gets pulled into the validation command.  For now, changing this print is actually an API change and we should hold off until the feature for this is built.

Reply:
Makes sense! Will hold on for these!